tests/integration/markers.py

```diff
@@ -11,6 +11,8 @@
 only_without_juju_secrets = pytest.mark.skipif(
     juju_.has_secrets, reason="Requires juju version w/o secrets"
 )
+only_with_juju_3 = pytest.mark.skipif(not juju_.is_juju_3, reason="Requires juju version >= 3")
```

Comment:
At the moment this is equivalent to `only_with_juju_secrets`. Do we need a separate marker here?